Gem/Settings/Scenes/ChainListSettings.swift

```diff
@@ -9,25 +9,42 @@ import Style
 struct ChainListSettingsScene: View {
 
     @Environment(\.nodeService) private var nodeService
+
     let model = ChainListSettingsViewModel()
-    
+    @State private var searchQuery = ""
+
     var body: some View {
         List {
-            ForEach(model.chains) { chain in
+            ForEach(model.items(for: searchQuery)) { chain in
                 NavigationLink(value: chain) {
                     ChainView(chain: chain)
                 }
             }
         }
+        .overlay {
```

Comment:
is there other options besides overlay?

Reply:
There are many options to ingrate empty state. The common and modern option to integrate it via overlay, which are best practise for new modern apps iOS16/17+. Like here, https://www.avanderlee.com/swiftui/contentunavailableview-handling-empty-states/ . I defiantly suggest to use it via overlay in that example